### ISSUE TITLE
Regression - cssprop no longer generated in json

### ISSUE THREAD

**Issue by MatthieuLebigre** [runem/web-component-analyzer#84]
Hi !
Thanks for your tool, it's really appreciated !
I've seen a regression in version 0.1.12 : cssprop are no longer generated in json output. It's working in markdown output thought.
It was working in 0.1.11.

Thanks

**Comment by runem** [comment]
Hi, thank you so much for opening an issue!

First of all, I will have to see an example/repro where it doesn't pick up your cssprops, because the following example works for me:
```
/**
 * @cssprop --this-is-a-css-prop - Here is a comment
 */
 class MyElement extends HTMLElement { 
 }

 customElements.define("my-element", MyElement);
```

I'm really sorry if there has been a regression! I'll make sure to fix it as soon as possible when I can reproduce it :tada:

**Comment by MatthieuLebigre** [comment]
Thanks for the quick answer !

But there is something I don't understand.

I try your exact same code in my project, and it doesn't work. Here's the output:

```
{
  "version": 2,
  "tags": [
    {
      "name": "my-element",
      "jsDoc": "/**\r\n * @cssprop --this-is-a-css-prop - Here is a comment\r\n */",
      "attributes": [],
      "properties": [],
      "events": [],
      "slots": []
    }
  ]
}
```

If I switch to the markdown output, it works fine ...
Any clue ?

**Comment by runem** [comment]
Oh! Right now the JSON format is in the experimental state. Up until now I just converted the internal data structure to JSON and this was why the format contained all possible information. The JSON format is still being developed and will eventual have breaking changes. I also think it will get "cssprops" in the future, but not right now. 

Would it work for you if I introduced a "debug" format that outputs the same format as you were used to? I'm not sure if I can promise this format not to have breaking changes in the future, but you would always be able to get all possible information from your elements, including cssprops :-)
